api-specs/api_spec_PGS.yaml

```diff
@@ -107,39 +139,6 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  /request-payments/bancomatpay/{requestId}:
-    get:
-      tags:
-        - BancomatPay
-      operationId: retrieve the correlationId of a BancomatPay payment request
-      parameters:
-        - in: path
-          name: requestId
-          schema:
-            type: string
-            format: uuid
-          required: true
-          description: PGS-generated GUID of the request to retrieve
-          example: 77e1c83b-7bb0-437b-bc50-a7a58e5660ac
-      responses:
-        "200":
-          description: OK
-          content:
-            '*/*':
-              schema:
-                $ref: '#/components/schemas/BPayInfoResponse'
-        "404":
-          description: Not found
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/BPayInfoResponse'
-        "500":
-          description: Internal server error
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/BPayInfoResponse'
```

Comment:
Why was this removed?

Reply:
it's not been removed, it's been moved up under /request-payments/bancomatpay (because the path param has been converted to a query param)